flights/berks_ceh_insurances_flight.yaml

```diff
@@ -12,27 +12,38 @@ entityDefinitions:
         type: "ol.datasource"
         transforms:
         - !<transforms.ValueTransform>
-          value: "Berks County CEH"
+          value: "Berks CEH"
     conditions:
     - !<conditions.BooleanIsNullCondition>
       column: "PersonalID"
       reverse: true
     name: "client"
 
 
-  medicaid: 
+  medicaid:
     fqn: "ol.insurance"
     entitySetName: "BerksCEHMedicaid"
-    propertyDefinitions: 
-      general.id: 
+    propertyDefinitions:
+      general.id:
         type: "general.id"
-        column: "IncomeBenefitsID"
+        transforms:
+        - !<transforms.ConcatTransform>
+          columns: ["Medicaid", "NoMedicaidReason"]
+          separator: "_"
+      general.status:
+        type: "general.status"
+        column: "Medicaid"
+        transforms:
+        - !<transforms.ParseIntTransform> {}
+        - !<transforms.ReplaceTransform>
+          target: ["0", "1"]
+          goal: ["No", "Yes"]
       ol.organizationname:
         type: "ol.organizationname"
         transforms:
-        - !<transforms.ValueTransform> 
+        - !<transforms.ValueTransform>
           value: "Medicaid"
-      ol.notes: 
+      ol.notes:
```

Comment:
I think the property ol.statusdetails would be better to use here instead of ol.notes, and in all of the insurance entity sets - you think?

Reply:
that makes sense - done